parser/parse_test.go

```diff
@@ -62,6 +62,10 @@ func TestRecursive(t *testing.T) {
 	if foundC {
 		t.Error("Did not expect to find package c")
 	}
+	// only has three package: package "a", package "b", and package "" for all
+	if len(findTypes) != 3 {
```

Comment:
I was expecting a check that the type identified is actually correct? From my reading the wrong code was still making a type, it's just not the right construction.